controller/node_controller.go

```diff
@@ -463,8 +463,7 @@ func (nc *NodeController) syncNode(key string) (err error) {
 	}
 
 	// sync default disks on labeled Nodes
-	node, err = nc.syncDefaultDisks(node)
-	if err != nil {
+	if err = nc.syncDefaultDisks(node); err != nil {
```

Comment:
Should be merged with the first commit?

Reply:
This commit is used to fix the existing NPE issue. The 2nd commit is used to implement the feature. I think it's better not to merge these 2 commits.

Reply:
As long as it's not fixing the issue introduced by the first commit, it's fine to keep it like this for this PR. Next time better put the fix of original issue first.